full-service/Cargo.toml

```
@@ -70,6 +74,7 @@ grpcio = "0.12"
 hex = { version = "0.4", default-features = false }
 libsqlite3-sys = { version = "0.26", features = ["bundled-sqlcipher"] }
 num_cpus = "1.16"
+prost = "0.12"
```

Optional/Nit: `Cargo.lock` shows the rest of the project uses `prost 0.11.9`, so using 0.12 here winds up with 2 versions of that lib.
```suggestion
prost = "0.11"
```